atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/HikariCPConnectionManager.java

```diff
@@ -121,34 +127,25 @@ private Connection acquirePooledConnection(long startMillis) throws SQLException
         }
     }
 
-    private void logConnectionAcquisitionTiming(long startMillis) {
-        long elapsedMillis = System.currentTimeMillis() - startMillis;
-        if (elapsedMillis > 1000) {
-            log.warn("Waited {}ms for connection", elapsedMillis);
-            logPoolStats();
-        } else {
-            log.debug("Waited {}ms for connection", elapsedMillis);
-        }
-    }
-
     private void logPoolStats() {
         if (log.isTraceEnabled()) {
             State stateLocal = state;
             HikariPoolMXBean poolProxy = stateLocal.poolProxy;
             if (poolProxy != null) {
                 try {
                     log.trace(
```

Comment:
Does this still make sense at trace log?

Reply:
Yep, this was changed to warn in the 2nd PR after some discussion #3589